docs/website/docs/hub/intro.md

```diff
@@ -3,41 +3,120 @@ title: Introduction
+* Visual dashboards for pipeline health and run history
+
+### Runtime
+
+**Runtime** - a managed cloud runtime operated by dltHub:
+
+* Scalable execution for pipelines and transformations
+* APIs, web interfaces, and auxiliary services
+* Secure, multi-tenant infrastructure with upgrades and patching handled for you
+
+:::tip
+Prefer full control? See [Enterprise](#tiers--licensing) below for self-managed options.
+:::
+
+### Storage
+
+**[Storage](ecosystem/iceberg.md)**. Choose where your data lives:
+
+* Managed lakehouse: Iceberg open table format (or DuckLake) managed by dltHub
+* Bring your own storage: connect to your own lake/warehouse when needed
+
+## Tiers & licensing
+
+Some of features described in this documentation are free to use. Others require a paid plan. Latest pricing & full feature matrix can be found live on our website.
+Most features support a self-guided trial right after install, check the [installation instructions](getting-started/installation.md) for more information.
+
+| Tier                  | Best for                                                                                   | Runtime                   | Typical use case                                                             | Notes                                          |
+| --------------------- | ------------------------------------------------------------------------------------------ | ------------------------- |------------------------------------------------------------------------------| ---------------------------------------------- |
+| **dltHub Basic**      | Solo developers or small teams owning a **single pipeline + dataset + reports** end-to-end | Managed dltHub Runtime    | Set up a pipeline quickly, add tests and transformations, share a simple app | Optimized for velocity with minimal setup      |
+| **dltHub Scale**      | Data teams building **composable data platforms** with governance and collaboration        | Managed dltHub Runtime    | Multiple pipelines, shared assets, team workflows, observability             | Team features and extended governance          |
+| **dltHub Enterprise** | Organizations needing **enterprise controls** or **self-managed runtime**                  | Managed or self-hosted Runtime | On-prem/VPC deployments, custom licensing, advanced security                 | Enterprise features and deployment flexibility |
+
+### Who is dltHub for?
+
+* Python developers who want production outcomes without becoming infra experts
+* Lean data teams standardizing on dlt and wanting integrated quality, transforms, and sharing
+* Organizations that prefer managed operations but need open formats and portability
 
-3. Do not forget to read our [EULA](EULA.md) and [Special Terms](EULA.md#specific-terms-for-the-self-issued-trial-license-self-issued-trial-terms)
-for self issued licenses.
+:::note
+* You can start on Basic and upgrade to Scale or Enterprise later, no code rewrites.
```

Comment:
Not sure about that tbh. I think Scale will be more opinionated on how to write code. It will be more declarative. But what we can do for sure: anything that works in OSS will work in dlthub without code changes.